.coveragerc

```diff
@@ -3,6 +3,7 @@ skip_covered = True
 show_missing = True
 fail_under = 75
 exclude_lines =
-    pragma: no cover
-    if __name__ == .__main__.:
     def __repr__
```

Comment:
**commit message:**
With the latest updates to this PR this specific commit subject is rather confusing as you're by no means increasing coverage, merely adding a few more patterns to ignore during the coverage calculation.